includes/class-wcpdf-admin.php

```diff
@@ -1402,7 +1404,22 @@ public function sort_orders_by_numeric_invoice_number( $query ): void {
 			return;
 		}
 
-		$query->set( 'orderby', 'meta_value_num' );
+		$query->set( 'orderby', $this->is_invoice_number_numeric() ? 'meta_value_num' : 'meta_value' );
+	}
+
+	/**
+     * Determines if the invoice number is numeric.
+     * It evaluates the presence of non-numeric characters in the prefix and suffix of the invoice number.
+     *
+	 * @return bool
+	 */
+	private function is_invoice_number_numeric() {
+		$invoice          = wcpdf_get_invoice( null );
+		$invoice_settings = $invoice->get_settings();
```

Comment:
Maybe you could use:

`$invoice_settings = WPO_WCPDF()->settings->get_document_settings( 'invoice' );`